compression/compression-multiple-chunks.any.js

```diff
@@ -0,0 +1,63 @@
+// META: global=worker
+// META: script=pako/pako_inflate.min.js
+// META: timeout=long
+
+'use strict';
+
+// This test asserts that compressing multiple chunks should work.
+
+// ('Hello', 3) => TextEncoder().encode('HelloHelloHello')
+function makeExpectedChunk(input, numberOfChunk) {
+  let expectedChunk = '';
```

Comment:
You can make this function shorter using `input.repeat(numberOfChunks)`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat.

Reply:
That's nice. Thanks!